### DISCUSSION TITLE
Certified GitOps Associate (CGOA) exam

### DISCUSSION THREAD

**Discussion by bradmccoydev** [open-gitops/project#156] [General]
The Certified GitOps Associate (CGOA) exam, is a joint collaboration between the Cloud Native Computing Foundation and the Continuous Delivery Foundation. It is being run by the Linux Foundation as an official example like the KCNA, CKA, CKAD, PCA etc.

The creation of the exam is in two phases as follows:

1. Create the blueprint of the example by doing market research and creating a Job Task Analysis.
2. Work with SME's to create the questions for the exam.

We have now finished the 1st phase of the exam and are ready to start building the questions.

In the first phase, we created the title, scope statement, target audience, domains, and competencies,  which the Linux Foundation has requested to be private for now. The purpose of this discussion is to make the exam as transparent as possible to ensure we can make the best exam for folks keeping to the Linux foundations wishes of not exposing too much information apart from the SME's that will be working on the questions.

We have a list of the SMEs now that have been identified and have some room for more. We are hoping to discuss this at Kubecon EU in Amsterdam. cc @scottrigby

**Comment by roberthstrand** [comment]
Sign me up @bradmccoydev. Are you coming to the OpenGitOps project meeting?

**Reply by bradmccoydev** [reply]
Hey, @roberthstrand sounds good, Yes I will be there!

**Comment by otkd** [comment]
I've put together a [Study Guide](https://github.com/otkd/CGOA-Study-Guide) for any other beta testers out there

**Comment by scottrigby** [comment]
🎉🎓 The CGOA exam and certification is complete available for anyone to take! - exam: https://training.linuxfoundation.org/certification/certified-gitops-associate-cgoa/
- Announcement: https://training.linuxfoundation.org/blog/cgoa-certification/

We need to promote this more:
- issue: Also https://github.com/open-gitops/website/issues/130
